kentikapi/internal/api_resources/my_kentik_portal_test.go

```diff
@@ -0,0 +1,228 @@
+package api_resources_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_connection"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_resources"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/utils"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTenantsList(t *testing.T) {
+	//arrange
+	getAllResponse := `
+	[
+		{
+			"id": 577,
+			"name": "test_tenant",
+			"description": "This is test tenant",
+			"users": [
+				{
+					"id": "148099",
+					"user_email": "test@tenant.user",
+					"last_login": null,
+					"tenant_id": "577",
+					"company_id": "74333"
+				},
+				{
+					"id": "148113",
+					"user_email": "user@testtenant.com",
+					"last_login": null,
+					"tenant_id": "577",
+					"company_id": "74333"
+				}
+			],
+			"created_date": "2020-12-21T10:55:52.449Z",
+			"updated_date": "2020-12-21T10:55:52.449Z"
+		},
+		{
+			"id": 578,
+			"name": "test_tenant2",
+			"description": "",
+			"users": [],
+			"created_date": "2020-12-21T10:57:53.425Z",
+			"updated_date": "2020-12-21T10:57:53.425Z"
+		}
+	]`
+
+	transport := &api_connection.StubTransport{ResponseBody: getAllResponse}
+	myKentikPortalAPI := api_resources.NewMyKentikPortalAPI(transport)
+
+	//act
+	tenants, err := myKentikPortalAPI.GetAll(nil)
+
+	//assert
+	assert := assert.New(t)
+	require := require.New(t)
+
+	require.NoError(err)
+	assert.Zero(transport.RequestBody)
+
+	assert.Len(tenants, 2)
```

Comment:
I think that it would be more readable to check response with single assert statement: `assert.Equal(expectedTenant, tenant)` 

In other tests too.

See: https://github.com/kentik/community_sdk_golang/blob/7ed66588c532214584238df5a8dba31e0c7b0e9b/kentikapi/client_integration_test.go#L170